idx/memory/memory.go

```diff
@@ -1190,7 +1192,7 @@ func (m *MemoryIdx) delete(orgId int, n *Node, deleteEmptyParents, deleteChildre
 func (m *MemoryIdx) Prune(orgId int, oldest time.Time) ([]idx.Archive, error) {
 	oldestUnix := oldest.Unix()
 	orgs := make(map[int]struct{})
-	if orgId == -1 {
+	if orgId == idx.OrgIdPublic {
```

Comment:
Doesnt look like the use of `-1` had anything to do with the PublicOrg.  So i think instead this should be 
```
if orgId == 0 {
```